### PR TITLE
no more error on all-invocations re-run

### PR DESCRIPTION
**Pull request by fladdimir**
As suggested in https://github.com/microsoft/vscode-java-test/pull/1412#pullrequestreview-949352726 , selecting all invocations of a method for re-run does not lead to a "not supported error" anymore. Instead, the method is run normally:

rerun_all_invocations_of_test_method

resolve https://github.com/microsoft/vscode-java-test/issues/1416

### CODE REVIEW

**Comment by jdneo** [comment]
Thank you! I'll start review the change after 0.35.0 is released.

**Comment by fladdimir** [comment]
> Thank you! I'll start review the change after 0.35.0 is released.

Sounds good, thanks!

File: src/controller/testController.ts [hunk 1]
```diff
@@ -311,12 +303,22 @@ function handleInvocations(testItems: TestItem[]): TestItem[] {
     // remove invocations if they are already included in selected higher-level tests
     testItems = testItems.filter((item: TestItem) => !(isInvocation(item) && isAncestorIncluded(item, testItems)));
```
Comment (jdneo) [inline src/controller/testController.ts:304]: Maybe put this line into the `mergeInvocations()` as well? Since it's dosing a kind of merging operations

**Review by jdneo — COMMENTED** [review]
Sorry for being late this time. I got a fever and was keeping rest days before. :)

Overall, the change looks good to me. Just a small comment, after that it should be good to be merged.

One more idea is that if we can add some tests for those utils. Do you have interest for doing that one? If not it's totally fine, I can do that later.

**Comment by fladdimir** [comment]
> Sorry for being late this time. I got a fever and was keeping rest days before. :)
> 
> Overall, the change looks good to me. Just a small comment, after that it should be good to be merged.
> 
> One more idea is that if we can add some tests for those utils. Do you have interest for doing that one? If not it's totally fine, I can do that later.

No worries at all, hope you are getting better!

As for the comment and adding some tests, I'd like to do it and I will find some time during the next days.

**Comment by jdneo** [comment]
Thank you @fladdimir! And you can address the comment and adding test in separate PRs. Since we have the pre-release channel. So, it's fine to roll out this fix first.

**Review by jdneo — APPROVED** [review]
LGTM. Thank you @fladdimir!

**Pull request merged**